app/types/ChallengeBuilder.tsx

```diff
@@ -26,11 +26,14 @@ export const buildChallenge = async (
     await firestoreCtrl.getName(challenge.uid);
 
     const challengeData: DBChallenge = {
-      challenge_name: challenge.challenge_name,
-      description: challenge.description,
+      caption: challenge.caption,
```

Comment:
Caption and challenge Description?

Reply:
challenge Description is here the title of the challenge which will be used to filter posts.